content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/consistent-list-from-cache.md

```diff
@@ -0,0 +1,11 @@
+---
+title: ConsistentListFromCache
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+<!--
+Allow the API server to serve consistent lists from cache.
+-->
+允许 API 服务器从缓存中提供一致的列表。
```

Comment:
I'm not sure what the current page says, but I'd leave “list” verbatim. It's an API operation.

Reply:
I will follow the [API concepts](https://kubernetes.io/zh-cn/docs/reference/using-api/api-concepts/#api-verbs):

```suggestion
允许 API 服务器从缓存中提供一致的 list 操作。
```